src/Language/PureScript/Docs/Render.hs

```diff
@@ -196,16 +201,16 @@ renderDeclaration (P.TypeClassDeclaration name args implies ds) title = do
 
   children = map renderClassMember ds
 
-  -- TODO: Comments for type class members
+  -- TODO: Comments for type class members?
```

Comment:
The question mark should have been there originally; it means 'decide if we even want these, and then act accordingly'. I'm leaning towards no, but don't feel strongly.